gradle-plugin/src/test/kotlin/com/google/devtools/ksp/gradle/ProcessorClasspathConfigurationsTest.kt

```diff
@@ -96,7 +96,7 @@ class ProcessorClasspathConfigurationsTest {
             """
                 kotlin {
                     jvm { }
-                    js { browser() }
+                    js(BOTH) { browser() }
```

Comment:
Can't figure out a way to pass compiler free arguments to kmp project with android plugin applied